meta-jsdelivr/recipes-jsdelivr/jsdelivr-service/files/jsdelivr-docker-controller.py

```
@@ -0,0 +1,839 @@
+    return success, error_msg
+
+
+@app.route('/containers', methods=['GET'])
+def get_containers():
+    """
+    Returns the raw output of docker ps --all --no-trunc --format json
+    """
+    try:
+        # Execute docker ps command
+        result = subprocess.run(
+            ['docker', 'ps', '--all', '--no-trunc', '--format', 'json'],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)